include/dlaf/blas/scal.h

```diff
@@ -0,0 +1,26 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <complex>
+
+/// @file
+
+namespace blas {
```

Comment:
Should we be adding things to `namespace blas`? I see now that it's been done already for the `operator<<` overloads for `blas` types, but there's it's a necessary evil because of ADL. Are we sure these are not going to conflict with something from `blaspp` in the future?

Reply:
It is required.
`scal` has 6 variants (s, d, c, z, cs and zd), but only 4 are currently implemented by blaspp.
My use case need (depending on T) the s, d, cs and zd variants. So I need them to be in the same namespace.